lib/install/validate/storage.go

```diff
@@ -41,11 +40,6 @@ func (v *Validator) storage(ctx context.Context, input *data.Data, conf *config.
 		return
 	}
 
-	// provide a default path if only a DS name is provided
-	if imageDSpath.Path == "" {
```

Comment:
After this, path adds nothing special, so we get something like (in vsphere format) `[datastore] /VIC/portlayer-provided-UUID/images/imageName` -- the path as far as the validator is concerned is empty.